libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog.Unit.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+namespace EnergyOrigin.ActivityLog.Unit.Tests;
+
+public class UnitTest1
+{
+    [Fact]
+    public void Test1()
+    {
+        Assert.True(true);
```

Comment:
Delete?

Reply:
Build errors without test project. Unit tests will be added in next part.